plugins/updater/src/error.rs

```diff
@@ -62,6 +62,8 @@ pub enum Error {
     /// Temp dir is not on same mount mount. This prevents our updater to rename the AppImage to a temp file.
     #[error("temp directory is not on the same mount point as the AppImage")]
     TempDirNotOnSameMountPoint,
+    #[error("binary for the current target not found in the archive")]
+    BinaryNotFoundInAcrhive,
```

Comment:
There is a typo in this variant, 'Achrive' should be 'Archive'

Reply:
damn, thanks for catching that!
https://github.com/tauri-apps/plugins-workspace/pull/812